project-base/app/config/parameters_test.yml.dist

```diff
@@ -12,3 +12,5 @@ parameters:
     test_mailer_host: smtp-server
     test_mailer_user: ~
     test_mailer_password: ~
+
+    shopsys.content_dir_name: 'content-test'
```

Comment:
it seems little weird that you `shopsys.content_dir_name` is overwritten in `parameters_test.yml.dist` but defined in  `paths.yml`, should not it be rather in `parameters.yml.dist`? Have you discussed the solution with anybody?

Reply:
Yes I did. With our Architect :) Our paths.yml is kind of confusing filename because when you will open it, you will see another definition of another `parameters`. And every parameter that needs to be overridden during the tests, is prepared in parameters_test.yml.dist.